### DISCUSSION TITLE
difference between ale_cpp_clangtidy_options and ale_cpp_clangtidy_extra_options?

### DISCUSSION THREAD

**Discussion by laoshaw** [w0rp/ale#3760] [Q&A - Ask for help with problems]
I want to pass some -checks to ale and wonder which one is preferred?

let g:ale_cpp_clangtidy_options='-check=*' for example

**Comment by arthurkiller** [comment]
let g:ale_cpp_clangtidy_options="-*, -fuchsia-default-arguments-calls, -google-readability-casting, -llvm-header-guard, -llvm-else-after-return, -llvmlibc-*, -altera-*, -bugprone-easily-swappable-parameters, -cppcoreguidelines-macro-usage, -cppcoreguidelines-avoid-goto, -cppcoreguidelines-avoid-c-arrays, -cppcoreguidelines-avoid-magic-numbers, -cppcoreguidelines-special-member-functions, -cppcoreguidelines-pro-type-vararg, -cppcoreguidelines-pro-type-cstyle-cast, -cppcoreguidelines-pro-type-reinterpret-cast, -cppcoreguidelines-prefer-member-initializer, -cppcoreguidelines-pro-bounds-pointer-arithmetic, -cppcoreguidelines-pro-bounds-array-to-pointer-decay, -cppcoreguidelines-avoid-non-const-global-variables, -cppcoreguidelines-pro-bounds-constant-array-index, -modernize-use-nodiscard, -modernize-use-trailing-return-type, -modernize-return-braced-init-list, -modernize-avoid-c-arrays, -misc-non-private-member-variables-in-classes, -hicpp-use-auto, -hicpp-avoid-goto, -hicpp-signed-bitwise, -hicpp-no-array-decay, -hicpp-special-member-functions, -hicpp-braces-around-statements, -readability-identifier-length, -readability-magic-numbers, -readability-use-anyofallof, -readability-else-after-return, -readability-implicit-bool-conversion, -readability-braces-around-statements, -readability-redundant-access-specifiers, -readability-function-cognitive-complexity"
